### ISSUE TITLE
Can't use as favicon

### ISSUE THREAD

**Issue by ibLeDy** [jnsdrtlf/django-feather#1]
Would love to be able to use the icons as favicons, at the moment is not possible (or I am doing it wrong).

I tested it like this

```html
{% load icon %}

<!DOCTYPE html>
<html lang="en">

  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="shortcut icon" href="{% icon 'lock' %}">
  </head>

</html>
```

Based on the usual way of doing it with `static`

```html
{% load static %}

<!DOCTYPE html>
<html lang="en">

  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="shortcut icon" href="{% static 'img/favicon.svg' %}">
  </head>

</html>
```

**Comment by vitellolabs** [comment]
`{% static 'img/favicon.svg' %}` populates the link to the svg file where is `{% icon 'lock' %}` renders the svg file content. Those are two different things.

**Comment by ibLeDy** [comment]
Yeah, I was just pointing out that in some use cases the _"static way"_ is necessary or more convenient, but I can't think of any use case apart from the favicon (and I know it is a weird one).

**Comment by 3j14** [comment]
@NotepadExpert is correct, `{% icon "lock" %}` would just return the raw `<svg> ... </svg>` XML tags. Serving the icons as static files would involve a lot of changes, but I think there is a better way. I could just convert the SVG content into a data URI with Base64 encoding. This way, browsers will render the value of the `href` attribute as if it was a file. I can implement an option for that

**Comment by 3j14** [comment]
Closed with 8991d36

You can use it like this:
```html
<link rel="icon" type="image/svg+xml" href="{% "coffee" data_uri=True %}" >
```

However, this will not work in Safari, but works perfectly fine in Chrome and Firefox.

It is already [live on PyPI](https://pypi.org/project/django-feather/0.3.0/)!
